test/controllers/home_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class HomeControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
+    get home_index_url
```

Comment:
if updating the routes above, don't forget to update this to read `get root_url`. We can also discuss more about integration tests as you start to get into it.